### PR TITLE
[qt] Make Close the default button on PPD and actived ond ESC

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

**Review by pastk — CHANGES_REQUESTED** [review]
It makes "Close" a default button for Enter press (before it was "Route from").

But it doesn't fix the original bug with Esc press or wndow close button click.

upd: qt6.2.4

**Comment by Ferenc-** [comment]
> It makes "Close" a default button for Enter press (before it was "Route from").
> 
> But it doesn't fix the original bug with Esc press or wndow close button click.
> 
> upd: qt6.2.4

True. How about now?

**Review by pastk — APPROVED** [review]
works great, thanks!

File: qt/place_page_dialog_common.cpp [hunk 1]
```diff
@@ -26,6 +28,7 @@ void addCommonButtons(QDialog * this_, QDialogButtonBox * dbb, bool shouldShowEd
 
   QPushButton * routeToButton = new QPushButton("Route To");
   routeToButton->setIcon(QIcon(":/navig64/point-finish.png"));
+  routeToButton->setAutoDefault(false);
```
Comment (biodranik) [inline qt/place_page_dialog_common.cpp:31]: Why these calls are needed? To avoid capturing "Enter"?

Comment (Ferenc-) [reply qt/place_page_dialog_common.cpp:31]: Yes, if the user doesn't move the cursor, then the enter would activate the first where the autodefault is not disabled.

**Review by biodranik — APPROVED** [review]

**Pull request merged**